testdata/repos/standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-2.ebuild

```diff
@@ -0,0 +1,6 @@
+DESCRIPTION="Ebuild with variable in LICENSE"
```

Comment:
Hmm, maybe it'd also worth testing that the check works over `+=` in addition to `=`.

```
LICENSE="foo"  # ok
LICENSE+=" ${bar}"  # not ok
```

Reply:
done.